pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-cluster/index.md

```diff
@@ -8,6 +8,8 @@ beta: false
 enterprise: false
 ---
 
+<p class="message--note"><strong>NOTE: </strong>When specifying the control-plane-endpoint, if the endpoint is a fully-qualified domain name (FQDN), we recommend you make that explicit by adding a period to the end. This helps avoid unnecessary DNS queries using search paths. For example, --control-plane-endpoint-host endpoint.example.com</code>.</p>
```

Comment:
Personally, I think it's better to put it ahead of step #1.

BUT, I think to avoid confusion we should make the initial reference to 'control-plane-endpoint' match the option name exactly (which is 'control-plane-endpoint-host'):

```suggestion
<p class="message--note"><strong>NOTE: </strong>When specifying the <code>control-plane-endpoint-host</code>, if the endpoint is a fully-qualified domain name (FQDN), we recommend you make that explicit by adding a period to the end. This helps avoid unnecessary DNS queries using search paths. For example, <code>--control-plane-endpoint-host endpoint.example.com.</code></p>
```

Reply:
There are already 2 notes in-line with step 1, and this is pretty important, so the rationale here is important note above the Step 1, before the user begins the steps. A note under the step might mean they execute the step, and then read the notes.